apps/ledger-live-desktop/src/renderer/modals/AddAccounts/steps/StepChooseCurrency.tsx

```diff
@@ -136,19 +140,26 @@ const StepChooseCurrency = ({ currency, setCurrency }: StepProps) => {
       klaytn,
     ],
   );
+
   const currencies = useMemo(() => {
     const currencies = (listSupportedCurrencies() as CryptoOrTokenCurrency[]).concat(
       listSupportedTokens(),
     );
-    const deactivatedCurrencies = Object.entries(featureFlaggedCurrencies)
-      .filter(([, feature]) => !feature?.enabled)
-      .map(([name]) => name);
+
+    const deactivatedCurrencies = mock
+      ? []
+      : Object.entries(featureFlaggedCurrencies)
+          .filter(([, feature]) => !feature?.enabled)
+          .map(([name]) => name);
+
```

Comment:
I don't think we need to ignore feature flagged coins, as they just need to be enabled for the same environment that playwright is running on, which should be only dev or staging (never runs on prod builds I guess), so it just means we need to have the feature flag enabled on dev or staging envs, which should be the case anyway before we want to run the UI tests for that coin.

Saying all that just so we can avoid using the `MOCK` env var from within the code :slightly_smiling_face: it seems we want to limit this as much as possible, see discussion here https://ledger.slack.com/archives/C0308JT8WS2/p1692373491532989?thread_ts=1691507827.835819&cid=C0308JT8WS2

Reply:
Tbf I just took it from this PR because I didn't want to deal with merge conflicts later: https://github.com/LedgerHQ/ledger-live/pull/4353/files#diff-e05343023abba025ba4898ebc9f04f698c171b29d3efdb2fcbb07c95545bf98e

I can remove though. Is it okay @hzheng-ledger ?

Reply:
@haammar-ledger What you are saying is correct. We can remove this piece of code in theory.
I've just checked the firebase configuration. It seems that in the project "ledger-live-development" all the currencies in dev environment are enable.
but when we launch the playwright test, it uses the configuration of "ledger-live-testing" project, all the currencies flags are missing there. Therefore, it takes the default value which is false and the evm coins are not enabled. It causes evm playwright test case failure.
So I suggest we add the currencies feature flags in "ledger-live-testing" project firstly and remove this piece of code.